src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -137,6 +137,21 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
                         deployment.Annotations.Add("dapr.io/enabled", "true");
                         deployment.Annotations.Add("dapr.io/id", project.Name);
                         deployment.Annotations.Add("dapr.io/port", (httpBinding.Port ?? 80).ToString(CultureInfo.InvariantCulture));
+
+                        if (!config.Data.Any())
+                        {
+                            continue;
+                        }
```

Comment:
nitpick: this `if` isn't really doing anything useful. If the dictionary is empty both of the following blocks will be skipped anyway because they won't find the value they're looking for.

Reply:
I will remove it